src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -258,27 +335,24 @@ export class AllRecords extends React.Component<IAllRecordsProps, IAllRecordsSta
             </div>
           </div>
           <div className="sort-container">
-            {isMapView ? (
-              <div className="button-pill" onClick={this.toggleMapView}>
-                <span>
-                  <FontAwesomeIcon icon="th" />
-                  <MediaQuery minDeviceWidth={768}>
-                    &nbsp;
-                    {translate('providerSite.gridView')}
-                  </MediaQuery>
-                </span>
-              </div>
-            ) : (
-              <div className="button-pill" onClick={this.toggleMapView}>
+            <div className="button-pill" onClick={this.toggleMapView}>
+              <span>
+                <FontAwesomeIcon icon={isMapView ? 'th' : 'map'} />
+                <MediaQuery minDeviceWidth={MOBILE_WIDTH_BREAKPOINT}>
+                  &nbsp;
+                  {translate(isMapView ? 'providerSite.gridView' : 'providerSite.mapView')}
+                </MediaQuery>
+              </span>
+            </div>
+            <MediaQuery minDeviceWidth={MOBILE_WIDTH_BREAKPOINT}>
```

Comment:
If it is minDeviceWidth and this button is meant for desktop I think there should be DESKTOP_WIDTH_BREAKPOINT (I had the wrong value in the previous version)?